### DISCUSSION TITLE
abs-esp-brake-fundamentals/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#16] [Ideas]
# abs-esp-brake-fundamentals/

很多人"改车"缺乏对于车辆的基本认知，许多 “改装店” 的师傅/老板可能连制动力平衡都不太了解，每次问到如果改了前刹之后如何保证制动力平衡相关的问题的时候总是会有类似如下的语句来搪塞，比如：

https://nova.moe/abs-esp-brake-fundamentals/

**Comment by smalloldowl** [comment]
期待大佬的更新，我脑子里的小人已经在叫我赶紧还原刹车了。即使我经过了一些简单计算，但因为太多要素和数据，普通玩家根本无法获得并计算，所以就显得改刹车这件事，尤其是在街道上使用，变得让人有点不安…

**Reply by n0vad3v** [reply]
计算参数这个确实，感觉普通玩家要**比较**精确的获得改后的前后刹车盘/卡钳尺寸只能参考同车型的高性能版本作为大致参考（比如十代思域参考 FK8 ，普通高尔夫参考高尔夫R），不然就只能根据经验判断，然后还要祈祷原车 ABS 标定不要因为改了刹车出现抽风的情况（新 BRZ 似乎就是一个典型，听到过很多改了卡钳之后重刹 ABS 顶脚的反馈）

而根据经验的话又需要精确，同场地，尽量同工况的的制动距离测量来判断制动比例变化是否影响制动距离，以及细腻循迹刹车时的车身动态感知...😇

当然，如果只是真日常开开，或许还好 😂 也就是许多改装店的话术（比如「前刹车本来就承担 80% 的制动力，只改前刹没有影响」），毕竟大部分消费者没有以上经验和验证的能力+想法。

**Reply by smalloldowl** [reply]
我是对比了改装前后的：活塞面积*轴心到刹车片中心距离*μ。因为我只改前轮走街要外观，所以我假定数据变化别太大就能大致维持（最大刹车踏板行程下的）前后刹车比，但是市面上刹车片的μ就如同刮刮乐，真想不通国内改刹车是谁带起来的…

**Comment by mx5typeRS** [comment]
谢谢楼主的分享，特意注册帐号来回复。关于你评论区提到的abs顶脚问题，我自己学习研究后总结主要是通过不正确的刹车改装然后和原车abs程序不匹配所引起的，常见于赛道（或100-0距离测试），欧美的环境里有专用术语叫做ice mode。我也是看了教主在b站的视频了解到这个问题，很可惜在简体中文网络资源内很难再找到更多关于这个问题的讨论，毕竟国内懂车爱车还爱开车下赛道的人实在太少了，因此只能再去西方论坛里翻别的车型的赛道经验。

像86 brz这个车型主要是fa20的第一代会出现abs顶脚，二代fa24的倒还好。在此我也推荐一个关于刹车配比的网站，在知道卡钳活塞尺寸和刹车盘面积的前提下可以通过这个网站去模拟不同卡钳搭配的前后配比，比如原厂的brz无论是前2后1的原厂刹车还是斯巴鲁搭配好的前后brembo刹车，在这个系统里的配比数值都是很精确地在0.67左右，因此理论上来说无论只选择前面还是选择前后套件，尽量和原厂配比接近就不太会出现abs有关的问题，英国的gr86 forum里也有懂车网友总结了各类刹车卡钳参数并直接提醒你是否会破坏原车的bias，国内么...屈指可数...大部分也就知道个4活塞6活塞，所以关于你提到的国内乱改卡钳+大部分改装店根本不懂技术真的是太典了，不能再同意更多。

**Reply by mx5typeRS** [reply]
网站 http://track-build.com/brake-bias-calc

**Reply by n0vad3v** [reply]
感谢专程注册一个程序员居多的平台来评论 ❤️ 现在使用静态博客系统一直苦于没有通用的且不会严格审查的留言平台，目前用的 Giscus 属于体验还行的，缺点就是依赖 GitHub，导致目前来留言的用户基本都是程序员 😅 能因为自己的一点分享吸引/帮助到和我有类似想法和调研能力的朋友，感觉很欣慰

http://track-build.com/brake-bias-calc 这个网站确实不错诶，还是第一次见到这种可以相对定量的测量的网站，感觉对于上面 @smalloldowl 应该也有不少帮助，有时间我也研究研究，说不定可以收集一些常见车型数据然后做一个表格方便大家参考的样子。

国内的改装感觉民科居多，既有从业者的原因，也有玩家自身的原因（然后就形成了一个负反馈），很难找到一个合理的讨论严谨（或者说相对科学/有据可参）的改装讨论（为数不多的见到的还是在 RedblazeUltra 的评论区，例如 https://www.bilibili.com/video/BV1hz4y1s7aE/ ）
